v2/AdminCore.Services/EventService.cs

```diff
@@ -74,10 +74,15 @@ public IList<EventDto> GetEventsByEmployeeId(int employeeId, EventTypes eventTyp
 
     public IList<EventDateDto> GetEventDatesByEmployeeAndStartAndEndDates(DateTime startDate, DateTime endDate, int employeeId)
     {
-      var eventDates = DatabaseContext.EventDatesRepository.Get(x => x.StartDate >= startDate
-                                                                     && x.EndDate <= endDate
-                                                                     && x.Event.EmployeeId == employeeId);
-      return _mapper.Map<IList<EventDateDto>>(eventDates);
+      endDate = SetEndDateToEndOfDay(endDate);
```

Comment:
Bit messy, rather than adding hours and minutes and using <=, add 1 day and use <